### PR TITLE
service registry management SD

### PR DESCRIPTION
**Pull request by rbocsi**
also minor fix

### CODE REVIEW

File: src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex [hunk 1]
```diff
@@ -0,0 +1,1045 @@
...
+\end{table}
+
+\msubsection{struct}{ServiceDefinitionRemoveRequest}
+\label{sec:model:ServiceDefinitionRemoveRequest}
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{4.1cm} | p{4.8cm} | p{2cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Mandatory & Description \\ \hline
+authentication & \hyperref[sec:model:Identity]{Identity} & yes & The requester of the operation. \\ \hline
+names &  \pref{List}$<$\pref{Name}$>$ & yes & Names of the service definitions that need to remove. \\ \hline
```
Comment (Katinka666) [inline src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex:726]: ` that need to remove` ->  that need to **be removed**

File: src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex [hunk 2]
```diff
@@ -0,0 +1,1045 @@
...
+\end{table}
+
+\msubsection{struct}{ServiceRemoveRequest}
+\label{sec:model:ServiceRemoveRequest}
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{4.1cm} | p{4.8cm} | p{2cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Mandatory & Description \\ \hline
+authentication & \hyperref[sec:model:Identity]{Identity} & yes & The requester of the operation. \\ \hline
+instances &  \pref{List}$<$\pref{Name}$>$ & yes & Identitifers of the service instances that need to remove. \\ \hline
```
Comment (Katinka666) [inline src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex:881]: `need to remove`  -> need to **be removed**

File: src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex [hunk 3]
```diff
@@ -0,0 +1,1045 @@
...
+\clearpage
+
+\msubsection{struct}{InterfaceTemplateRemoveRequest}
+\label{sec:model:InterfaceTemplateRemoveRequest}
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{4.1cm} | p{4.8cm} | p{2cm} | X |} \hline
+\rowcolor{gray!33} Field & Type & Mandatory & Description \\ \hline
+authentication & \hyperref[sec:model:Identity]{Identity} & yes & The requester of the operation. \\ \hline
+names &  \pref{List}$<$\pref{InterfaceTemplate}$>$ & yes & Names of the interface templates that need to remove. \\ \hline
```
Comment (Katinka666) [inline src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex:986]: `need to remove` -> need to **be removed**

File: src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex [hunk 4]
```diff
@@ -0,0 +1,1045 @@
...
+\label{sec:model:primitives}
+
+Types and structures mentioned throughout this document that are assumed to be available to implementations of this service.
+The concrete interpretations of each of these types and structures must be provided by any IDD document claiming to implement this service.
+
+\begin{table}[ht!]
+\begin{tabularx}{\textwidth}{| p{3cm} | X |} \hline
+\rowcolor{gray!33} Type & Description \\ \hline
+\pdef{Address}          & A string representation of the address. \\ \hline
+\pdef{AddressType}      & Any suitable type chosen by the implementor or service. \\ \hline
```
Comment (Katinka666) [inline src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex:1000]: I think you meant implementor **of the** service (or **of** service)

File: src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex [hunk 5]
```diff
@@ -0,0 +1,1045 @@
...
+\pdef{DateTime}         & Pinpoints a specific moment in time. \\ \hline
+\pdef{Direction}        & The direction of a sorting operation. Possible values are the representation of ascending or descending order. \\ \hline
+\pdef{ErrorType}        & Any suitable type chosen by the implementor of service. \\ \hline
+\pdef{InterfaceTemplate} & A string identifier of an interface descriptor. \\ \hline
+\pdef{List}$<$A$>$      & An \textit{array} of a known number of items, each having type A. \\ \hline
+\pdef{LogSeverity}      & The kind and seriousness of a log message. \\ \hline
+\pdef{Name}             & A string identifier that is intended to be both human and machine-readable. \\ \hline
+\pdef{Number}           & Decimal number. \\ \hline
+\pdef{Object}           & Set of primitives and possible further objects. \\ \hline
+\pdef{OperationStatus}  & Logical, textual or number value that indicates whether an operation is a success or a failure. Multiple values can be used for success and error cases to give additional information about the nature of the result. \\ \hline
```
Comment (Katinka666) [inline src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex:1011]: I think "Logical, textual or **numerical** value" is better

File: src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex [hunk 6]
```diff
@@ -0,0 +1,1045 @@
...
+\pdef{PropertyValidator} & An identifier of any suitable validtor function chosen by the implementor of service. \\ \hline
```
Comment (Katinka666) [inline src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex:1012]: typo: validator

**Review by Katinka666 — CHANGES_REQUESTED** [review]

File: src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex (hunk 4 above)
Comment (rbocsi) [reply src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management.tex]: I Also had to fix it in some previous documents.

File: src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management_sd.tex [hunk 7]
```diff
@@ -136,7 +140,7 @@ \section{Overview}
 In Section \ref{sec:model}, we end the document by presenting the data types used by the mentioned operations.
 
 \subsection{How This Service Is Meant to Be Used}
-The service's purpose is to handle the various entities of the Service Registry centrally and in bulk. Such entities are devices, systems, service definitions, service instances and interface templates. This approach makes possible that the individual application systems do not have to anything to work within the Local Cloud.
+The service's purpose is to handle the various entities of the Service Registry centrally and in bulk. Such entities are devices, systems, service definitions, service instances and interface templates. This approach makes possible that the individual application systems do not have to do anything to able to work within the Local Cloud.
```
Comment (Katinka666) [inline src/serviceregistry/5_0_0/sd/service-registry-management/service-registry-management_sd.tex:143]: `to able to work` -> to **be** able to work

**Review by Katinka666 — APPROVED** [review]

**Review by borditamas — APPROVED** [review]

**Pull request merged**